### ISSUE TITLE
Dexterity needed for plone4.3.7

### ISSUE THREAD

**Issue by maartenkling** [plone/plone.app.linkintegrity#29]
https://github.com/plone/plone.app.linkintegrity/commit/ac62c62c218d3d9be0897e03db911d1896bba97d#diff-7fd571a41987966fc8c863aa290b99baR12

Now we need dexterity in plone 4.3.7, my tests fail

ZopeXMLConfigurationError: File "/blabla/eggs/Products.CMFPlone-4.3.7-py2.7.egg/Products/CMFPlone/configure.zcml", line 25.2-25.47
    ZopeXMLConfigurationError: File "/blabla/eggs/plone.app.linkintegrity-1.5.6-py2.7.egg/plone/app/linkintegrity/configure.zcml", line 38.4-43.38
    ImportError: No module named dexterity.interfaces

**Comment by frisi** [comment]
hi @maartenkling 
had the same error on an older plone 4.1 buildout.
i was able to fix this by changing my buildout configuration

from
```
eggs = 
    Products.CMFPlone
    Pilow
```

to
```
eggs = 
    Plone
    Pilow
```

so the Plone metapackage includes dexterity and Products.CMFPlone does not

**Comment by maartenkling** [comment]
@frisi thats also strange.

i'm using: 
[buildout]
extends =
    https://raw.github.com/collective/buildout.plonetest/master/test-4.x.cfg

no eggs = defined

**Comment by saily** [comment]
same issue here.

**Comment by saily** [comment]
btw. you can quick-n-dirty-fix this by adding `plone.dexterity` to your modules's test section in `setup.py`.

**Comment by mauritsvanrees** [comment]
Thanks for fixing, @pbauer. I have released 1.5.7.
